app/src/main/res/values/colors.xml

```diff
@@ -46,4 +46,11 @@
     <!--    Background-->
     <color name="app_bg">#e0e0e0</color>
     <color name="app_bg_light">#90A4AE</color>
+
+    <!-- YIM Colors-->
+    <color name="yim_150">#FFF80729</color>
```

Comment:
Aren't we using Jetpack Compose for the colors?

Reply:
My bad left it here by mistake. Was trying some stuff didn't work out.